kie-drools-wb/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/client/perspectives/DroolsAuthoringNoContextNavigationPerspective.java

```diff
@@ -77,37 +81,38 @@
     public void init() {
         explorerMode = ( ( Window.Location.getParameterMap().containsKey( "explorer_mode" ) ) ? Window.Location.getParameterMap().get( "explorer_mode" ).get( 0 ) : "" ).trim();
         projectPathString = ( ( ( Window.Location.getParameterMap().containsKey( "path" ) ) ? Window.Location.getParameterMap().get( "path" ).get( 0 ) : "" ) ).trim();
-        projectEditorDisableBuild = Window.Location.getParameterMap().containsKey( "no_build" );
-    }
+        projectEditorDisableBuild = Window.Location.getParameterMap().containsKey("no_build");
 
-    @Perspective
-    public PerspectiveDefinition getPerspective() {
-        final PerspectiveDefinitionImpl perspective = new PerspectiveDefinitionImpl( MultiListWorkbenchPanelPresenter.class.getName() );
-        perspective.setName( constants.project_authoring() );
+        final PlaceRequest placeRequest = generateProjectExplorerPlaceRequest();
+
+        docks.setup("AuthoringPerspectiveNoContext", placeRequest);
 
-        final PanelDefinition west = new PanelDefinitionImpl( SimpleWorkbenchPanelPresenter.class.getName() );
-        west.setWidth( 400 );
+    }
+
+    private PlaceRequest generateProjectExplorerPlaceRequest() {
         final PlaceRequest placeRequest = new DefaultPlaceRequest( "org.kie.guvnor.explorer" );
         if ( !explorerMode.isEmpty() ) {
             placeRequest.addParameter( "mode",
-                                       explorerMode );
+                    explorerMode );
         }
         if ( !projectPathString.isEmpty() ) {
             placeRequest.addParameter( "init_path",
-                                       projectPathString );
+                    projectPathString );
         }
         if ( projectEditorDisableBuild ) {
             placeRequest.addParameter( "no_build",
-                                       "true" );
+                    "true" );
         }
 
-        placeRequest.addParameter( "no_context",
-                                   "true" );
-
-        west.addPart( new PartDefinitionImpl( placeRequest ) );
+        placeRequest.addParameter("no_context",
+                "true");
+        return placeRequest;
+    }
 
-        perspective.getRoot().insertChild( CompassPosition.WEST,
-                                           west );
+    @Perspective
+    public PerspectiveDefinition getPerspective() {
+        final PerspectiveDefinitionImpl perspective = new PerspectiveDefinitionImpl( MultiListWorkbenchPanelPresenter.class.getName() );
+        perspective.setName( constants.project_authoring() );
```

Comment:
We've removed project explorer ("org.kie.guvnor.explorer")  and add it in the west dock.